src/sage/combinat/sloane_functions.py

```diff
@@ -603,7 +603,7 @@ def _eval(self, n):
         from sage.rings.lazy_series_ring import LazyPowerSeriesRing
         x = LazyPowerSeriesRing(ZZ, 'x').gen()
         p = 1/((1-x)*(1-x**2)*(1-x**5)*(1-x**10))
-        return ZZ(p.coefficient(n)
+        return ZZ(p.coefficient(n))
```

Comment:
Just `p[n]` would be better.  But using a lazy generating function to return a coefficient is a bit odd, why not do `return Partitions(n, parts_in=[1,2,5,10]).cardinality()`, this is much faster and also clearer.

Reply:
done using Partitions, but this seems to be dead slow...

Reply:
That's strange, can you show a comparison?  On my computer, using Partitions is much faster:
```
sage: a = sloane.A000008
sage: %timeit a[100000]
48.3 ms ± 2.25 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)
sage: %timeit Partitions(100000, parts_in=[1,2,5,10]).cardinality()
13.5 ms ± 198 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```

Reply:
Maybe it was the internal launch of gap that made it slow

Reply:
on macOS M1, I have similar behavior than @mantepse 
```py
sage: a = sloane.A000008
sage: %timeit a[100000]
26.1 ms ± 262 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
sage: %timeit Partitions(100000, parts_in=[1,2,5,10]).cardinality()
8.51 ms ± 309 µs per loop (mean ± std. dev. of 7 runs, 1 loop each)
```